src/credentials/FabricTable.cpp

```diff
@@ -51,6 +51,26 @@ constexpr TLV::Tag kFabricLabelTag = TLV::ContextTag(1);
 constexpr TLV::Tag kNextAvailableFabricIndexTag = TLV::ContextTag(0);
 constexpr TLV::Tag kFabricIndicesTag            = TLV::ContextTag(1);
 
+// Tagsd for commit marker storage
+constexpr TLV::Tag kMarkerFabricIndexTag = TLV::ContextTag(0);
+constexpr TLV::Tag kMarkerIsAdditionTag  = TLV::ContextTag(1);
+
+constexpr size_t CommitMarkerContextTLVMaxSize()
+{
+    // Add room for the future
```

Comment:
Is this the explanation for the `* 2` bit?  If so, it could use being a bit less cryptic....  But also, I'm not sure why adding room here "for the future" makes sense.

Reply:
Because otherwise you could revert into an app that has less metadata and blow-up on the read...

Reply:
I think clearly documenting that would be helpful.